history-of-quakers/modernized/27-addenda.adoc

```diff
@@ -21,6 +14,10 @@ And the original in Latin, soon after the first publication,
 being translated into English by B. F. was printed in the year 1663,
 from which translation take it as follows:
 
+[.embedded-content-document]
+--
+
+[.letter-heading]
 THE LIGHT UPON THE CANDLESTICK
```

Comment:
Let's remove the all caps here (even though it looks the same when rendered), if we ever change the styling of the letter-heading, we'd want this to be normal-cased, I think.